misc/handlers.go

```diff
@@ -23,7 +23,7 @@ import (
 
 func InitHandlers(rtr *mux.Router) {
 	rtr.HandleFunc("/robots.txt", handlerRobotsTxt)
-	rtr.HandleFunc("/static/style.css", handlerStyle)
+	rtr.HandleFunc("/static/style.css", HandlerStyle)
 	rtr.PathPrefix("/static/").
 		Handler(http.StripPrefix("/static/", http.FileServer(http.FS(static.FS))))
```

Comment:
Also you probably should move this stuff to the auth router as well, plus the favicon route. People may depend on static assets in their custom styles, and a favicon is generally not really private, given that we leak header links on auth fail already.

Reply:
I think I understand what you mean. I moved the routes to the auth router, but left the handler functions in misc.